app/EdgeOperations2d/ShortestEdgeCollapse.cpp

```diff
@@ -54,6 +54,7 @@ std::vector<TriMesh::Tuple> Edge2d::EdgeOperations2d::new_edges_after(
 
 bool Edge2d::EdgeOperations2d::collapse_shortest(int target_operation_count)
 {
+    size_t initial_size = get_vertices().size();
```

Comment:
Why is this here? This result is not used in your code.

Reply:
This is the old code. The whole /EdgeOPerations2d folder can be deleted soon